pkg/actor/validate_version.go

```diff
@@ -139,6 +139,10 @@ func (v *versionChecker) Act(ctx context.Context, cluster *resource.Cluster) err
 		err := WaitUntilJobPodIsRunning(ctx, clientset, job, log)
 		if err != nil {
 			log.Error(err, "job not found")
+			if dErr := deleteJob(ctx, cluster, clientset, job); dErr != nil {
```

Comment:
The log message does not make sense now

Reply:
updating the error on the previous line to be more accurate